cli/update.go

```diff
@@ -179,7 +179,7 @@ func download(version string) error {
 	if err != nil {
 		return err
 	}
-	url := fmt.Sprintf(downloadURL, version, strings.Title(runtime.GOOS))
+	url := fmt.Sprintf(downloadURL, version, strings.ToUpper(runtime.GOOS[:1])+runtime.GOOS[1:])
```

Comment:
`strings.Title()` is discouraged because of Unicode stuff, but I didn't feel like pulling in `/x/strings` just for this simple case.